projects/plugins/protect/readme.txt

```diff
@@ -1,68 +1,100 @@
 === Jetpack Protect ===
-Contributors: automattic,
-Tags: jetpack, stuff
+Contributors: automattic, retrofox, leogermani, renatoagds, bjorsch, ebinnion, fgiannar, zinigor, miguelxavierpenha, dsmart, jeherve, manzoorwanijk, njweller, oskosk, samiff, siddarthan, wpkaren, arsihasi, kraftbj, kev, sermitr, kangzj, pabline
+Tags: jetpack, protect, security, malware, scan
 Requires at least: 5.9
 Requires PHP: 5.6
 Tested up to: 6.0
-Stable tag: 0.1.0-alpha
+Stable tag: 1.0.0
```

Comment:
I would recommend  against uploading the stable tag here until we actually do the first release.

Reply:
I don't think we actually planned to add the tag here, but just have it in the readme.txt so that it will align correctly once we do the first release. Do you see this as problematic? I think otherwise we'd have to change this just before we do the actual release (and not forget it).

Reply:
> I think otherwise we'd have to change this just before we do the actual release (and not forget it).

Yes, that is what I would recommend. Our release scripts should help you with that.

Reply:
Okay then, I have changed that back.